frontend/App.tsx

```diff
@@ -70,7 +70,7 @@ export default function App() {
           <Stack.Screen name="HomeScreen" component={Homescreen} options={{ gestureEnabled: false, headerBackVisible: false }} />
           <Stack.Screen name="SettingsScreen" component={AccountSettings} />
           <Stack.Screen name="GameScreen" component={GameScreenFlow} options={{
-            headerTitle: () => { return <Text style={{color:"#5eafdf", fontSize:20}}>Vocabulary Practice</Text> },
+            headerTitle: () => { return <Text style={{ color: "#5eafdf", fontSize: 20 }}>Vocabulary Practice</Text> },
```

Comment:
Just leaving a note here that we'll probably have to update the fonts in a few files. Left a note on #196 (the fonts PR) as well. We can merge this PR in first and then handle the conflicts/update fonts in that PR since we're waiting on updating the fonts in the tutorial pictures anyway.

Relevant files: `App.tsx`, `PrimaryButton.tsx`, and `adminScreen.tsx`